### PR TITLE
Add luna moth wings to moths

### PR DESCRIPTION
**Pull request by boogiebogus**
## About the PR

Added a new wing customization choice for moths based on luna moths

## Why / Balance

Luna moths are a fairly famous kind of moth and I think it's weird that we don't have wings accurate to them. Named "Selene" because there's already wings named "Luna (Classic)" despite not really looking at all like luna moth wings.

## Technical details

Added YAML, sprites, and a few names in locale/

## Media

Pre-colored version (named "classic" to get the point of what it is across, even though that's not really accurate)
image
Recolorable version
image

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Nothing (i hope?)

**Changelog**

:cl: 
- add: added "Selene" wing customization option for moths

### CODE REVIEW

**Comment by Lyndomen** [comment]
Under Direction review, 24 hours!

File: Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json:1]: - 4 space indent
- dont mix tabs and spaces
- file needs newline at the end

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json (hunk 1 above)
Comment (boogiebogus) [reply Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json:1]: hopefully corrected now

Comment (deltanedas) [reply Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json:1]: change it to 4 spaces per indent instead of 2 and its good

Comment (boogiebogus) [reply Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json:1]: should be good now?

**Review by Lyndomen — APPROVED** [review]
Looking forward to seeing this in game!

**Review by deltanedas — APPROVED** [review]

**Pull request merged**